closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -946,34 +772,52 @@ boolean isRestrictedServiceAllowed(Service service) {
                 return false;
             } else if (constraints.length == 0) {
                 // Allow this provider with no constraints.
+                if (debug != null) {
```

Comment:
I see no reason why this file shouldn't be identical to later versions. Please add the debug code before line 772, the blank line before line 868 and update the comment on line 890.